tests/integration/suite/daprd/metrics/http/distributionbuckets/util.go

```diff
@@ -0,0 +1,22 @@
+package distributionbuckets
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+func getBucketFromKey(k string) (float64, error) {
```

Comment:
Consider passing in a `testing.T` so we can `require` inside the func, cleaning up the code a bit. 

```suggestion
func getBucketFromKey(t *testing.T, k string) float64 {
```